st2actions/st2actions/query/mistral/v2.py

```diff
@@ -107,19 +107,9 @@ def _get_workflow_tasks(self, exec_id):
         :type exec_id: ``str``
         :rtype: ``list``
         """
-        url = self._get_execution_tasks_url(exec_id)
-        resp = requests.get(url)
-        result = resp.json()
-        tasks = result.get('tasks', [])
-
-        result = []
+        wf_tasks = tasks.TaskManager(self._client).list(workflow_execution_id=exec_id)
```

Comment:
Looks like it's the same as what we were doing before. https://github.com/StackStorm/python-mistralclient/blob/master/mistralclient/api/v2/tasks.py#L30